theme/common.less

```diff
@@ -1033,6 +1041,22 @@ Field editors
                 padding-right: .5rem;
             }
         }
+
+        .extension-header {
+            font-family: @segoeUIFont;
+            width: 95%;
+            display: flex;
+            align-items: flex-end;
+            justify-content: space-between;
+        }
+
+        .import-button>.common-button {
+            padding: .8rem 1rem .95rem
+        }
+
+        .import-button>.common-button:hover {
+            filter: grayscale(.85) brightness(.75) contrast(1.3)
```

Comment:
common-buttons already have hover behavior. No need to overwrite it here. Same with the padding above; just use the default react-common styles

Reply:
When I use the default style the button looks like:
![image](https://user-images.githubusercontent.com/6496798/169897153-69116ff8-6904-4cd9-bec1-898f64c82637.png)
vs with padding:
![image](https://user-images.githubusercontent.com/6496798/169897723-9267487a-aa04-4da5-bfad-a516271f2d60.png)
which looks cleaner and close to the import button on the home page:
![image](https://user-images.githubusercontent.com/6496798/169897788-b2263a00-babb-46fc-bca5-68068b4c0eca.png)

Reply:
The filter was to make it even darker than the react-common button on hover to better match the behavior it has on the target home pages.